src/lib/ext-activitylog.js

```diff
@@ -33,26 +35,33 @@ class Model {
   matchLogWithFilterObj(log) {
     return (
       this.matchFilterId(log.id) &&
-      this.matchFilterViewType(log.viewType) &&
+      this.matchFilterViewType(log) &&
       this.matchFilterType(log.type) &&
+      this.matchFilterApiName(log) &&
       this.matchFilterKeyword(log.data)
     );
   }
 
-  matchFilterId(logId) {
-    return this.filter.id.has(logId);
+  matchFilterId(id) {
+    return this.filter.id.has(id);
   }
 
-  matchFilterViewType(logViewType) {
-    return this.filter.viewType.has(logViewType);
+  matchFilterViewType({ viewType, type }) {
+    return type === 'content_script'
```

Comment:
I suggest to use the following logic instead of a ternary.

```
if content script
  // Comment explaining why there is an early return.
  Return true
Return normal check
```

It is more readable, because it is more obvious that content scripts are excluded, together with the reason.

Reply:
done :-)